src/Entity/MarsScientistEntity.php

```diff
@@ -92,12 +93,20 @@ public function __construct(
         $this->station = $station;
     }
 
+    /**
+     * @param Collection $registredUsers
+     */
+    public function setRegistredUsers(Collection $registredUsers): void
+    {
+        $this->registredUsers = $registredUsers;
+    }
```

Comment:
1. Jak masz cos w jezyku to juz nie dodawaj PHPDOCa. W Landingach uzywamy PHPDoc jedynie do informacji o rzucanych wyjatkach.
2. unikaj setterów jezeli nie potrzebujesz ich
3. kolejnosc funkcji w klasie -> statyczne konstruktory, konstruktor, metody publiczne, metody prywatne